dev/devicelab/lib/framework/runner.dart

```diff
@@ -228,7 +229,8 @@ Future<TaskResult> runTask(
       'ext.cocoonRunTask',
       args: isolateParams,
       isolateId: result.isolate.id,
-    )).json!;
+    ))
+        .json!;
```

Comment:
nit: this doesn't look any better than before.

Reply:
Yeah, that was autoformat by the editor... i'll switch those back.

Reply:
removed all auto-format related changes.